source/MultiGrid/MultiGrid.js

```diff
@@ -72,23 +72,28 @@ export default class MultiGrid extends Component {
     rowIndex = 0
   } = {}) {
     const { fixedColumnCount, fixedRowCount } = this.props
-
-    this._bottomLeftGrid && this._bottomLeftGrid.measureAllCells({
-      columnIndex,
-      rowIndex: rowIndex - fixedRowCount
+    const col = (columnIndex >= fixedColumnCount ? columnIndex - fixedColumnCount : columnIndex)
+    const row = (rowIndex >= fixedRowCount ? rowIndex - fixedRowCount : rowIndex)
```

Comment:
Ah, fair enough. I think the correct fix is to add a `Math.max(0, maybeNegativeIndex)` in `MultiGrid` so that it doesn't pass negative values. As it is (with this change) we may skip cells in the bottom and/or right grids if indices > 0 and < fixed-count are passed.